crates/oxc_linter/src/rules/eslint/no_array_constructor.rs

```diff
@@ -64,6 +67,19 @@ impl Rule for NoArrayConstructor {
                 &new_expr.type_parameters,
                 false,
             ),
+            AstKind::ExpressionStatement(expr_stmt) => {
```

Comment:
This duplicates the `CallExpression` logic above, hence the duplicated error message.

Reply:
Oh of course! Thanks.